search-app/docs/api.md

```diff
@@ -1186,6 +1188,12 @@ Find collections that match all of the 'project' param values
 
    curl "%CMR-ENDPOINT%/collections?created_at\[\]=2000-01-01T10:00:00Z,2010-03-10T12:00:00Z&created_at\[\]=2015-01-01T10:00:00Z,"
 
+#### <a name="c-with-new-granules"></a> Find collections with new granules
+
+  Find collections containing granules added after a given date. The datetime has to be in yyyy-MM-ddTHH:mm:ssZ format.
+
+  curl "%CMR_ENDPOINT%/collections?has-granules-added-after=\[\]2015-01-01T10:00:00Z"
```

Comment:
How difficult would it be to support begin and end ranges just like the granules created_at API (and call the parameter `granules_created_at`)?